.env-example

```diff
@@ -1,7 +1,8 @@
-GH_APP_ID=' '
-GH_APP_INSTALLATION_ID=' '
-GH_APP_PRIVATE_KEY=' '
-GH_ENTERPRISE_URL=' '
-GH_TOKEN=' '
-INACTIVE_DAYS=365
-ORGANIZATION=' '
+ADDITIONAL_METRICS = ""
```

Comment:
Alphabetized!!!! ❤️

Reply:
I was self-reviewing the changes before committing and literally thought, "What would JM say in this PR?" and alphabetizing came to the top of my mind. 😂 You're training me well!